packages/database/src/migrations/20201014220808-AddLaosEocAggregateForecastMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+import { cloneDeep } from 'lodash'
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+
+const MAP_OVERLAY_GROUP_CODE = 'LA_EOC_Weather_Observations_And_Forecast';
+
+const commonMapOverlayConfig = {
+  userGroup: 'Laos EOC User',
+  dataElementCode: 'WTHR_FORECAST_PRECIP',
+  measureBuilderConfig: {
+    entityAggregation: {
+      dataSourceEntityType: 'city',
+    },
+  },
+  measureBuilder: 'sumAllPerOrgUnit',
+  presentationOptions: {
+    scaleType: 'neutral',
+    valueType: 'oneDecimalPlace',
+    displayType: 'spectrum',
+    measureLevel: 'City',
+    periodGranularity: 'day',
+    timePeriodEditable: false,
+  },
+  countryCodes: '{LA}',
+  projectCodes: '{laos_eoc}',
+};
+
+const FORECAST_MAP_OVERLAY_7_DAY = cloneDeep(commonMapOverlayConfig);
```

Comment:
I love `lodash.cloneDeep`! It may bit a bit heavy for front-end apps (since I believe it depends on multiple additional lodash modules), but for it's perfect for the back-end 👌

I can only imagine how useful it will be in other similar migrations!

Reply:
I just realised using it will make other migrations much simpler!